parent/pom.xml

```diff
@@ -293,9 +293,6 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-jar-plugin</artifactId>
-                    <configuration>
```

Comment:
Why?

Reply:
Jotm has third-party dependency that has another third-party dependency that is pulled from http repo. In 2.3.1-M1 version of Jotm this third party dependency was updated so build is not failing.